### DISCUSSION TITLE
[Enhancement] [date-picker] Date Range picker popup only one panel

### DISCUSSION THREAD

**Discussion by Vireakphol9** [element-plus/element-plus#16945] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `date-picker`

## Description

For daterange picker should add prop ( singlePanel: boolean ). Currently when we try to remove it by using set unlinkPanel="true" and hide second panel, the first panel not able to click to next month or year.

**Comment by cccMing** [comment]
我也有相同疑问，就区间选择 daterange 没有，不方便

**Comment by rzzf** [comment]
#19498 done.
